### DISCUSSION TITLE
regarding create_report function :

### DISCUSSION THREAD

**Discussion by aelsay96** [ansys/pyaedt#2445] [Q&A]
i worked on script to automate plots for my project 
here are example :
##############################
speed=["800rpm","900rpm","1000rpm"]
mean_para ={} 
mean_para["mean(Moving1.Torque)"]= "TorqueVsSpeedPlots"
variations1={}
variations1["Time"] =["0.12s","0.14s"]
variations1["spd"] =speed
for k, v in mean_para.items():
M2D.post.create_report(expressions=k, setup_sweep_name="", domain="sweep", variations=variations1,
primary_sweep_variable="spd",
report_category=None, plot_type="Rectangular Plot", context="Moving1", subdesign_id=None,
polyline_points=1001, plotname=v)
##############################
so this automate plot generation in Maxwell
**but the plot itself doesn't show any data except i open it and then hit new report button in Maxwell itself** 
this happen if i assign any variable except time as primary_sweep_variable

**Comment by Samuelopez-ansys** [comment]
@aelsay96 Please provide more information, and attach a dummy project to reproduce the issue. Thanks
